srcpkgs/python-distro/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'python-distro'
+pkgname=python-distro
+version=1.4.0
+revision=1
+wrksrc=distro-${version}
+build_style=python-module
```

Comment:
No new python2 code please! Since mate can work with python3, just add python3-distro.

Reply:
Fixed, thank you for the review. I will do the same with the tinyrss module in the cinnamon PR.